fe1-web/src/features/connect/screens/ConnectConfirm.tsx

```diff
@@ -66,9 +66,6 @@ const ConnectConfirm = () => {
     }
 
     const channel = getLaoChannel(laoId);
```

Comment:
this can now throw, right? what happens if it does?

Reply:
No it cannot 😅 In theory the laoId could be invalid but checking for that will require much more work throughout the codebase in various places

Reply:
ehm, I really do believe it can 😅

Reply:
Yes you are right.. 😂 Sorry for that!